core/modules/node/node.module

```diff
@@ -1953,96 +1979,6 @@ function node_get_recent($number = 10) {
 }
 
 /**
- * Page callback: Generates and prints an RSS feed.
- *
- * Generates an RSS feed from an array of node IDs, and prints it with an HTTP
- * header, with Content Type set to RSS/XML.
- *
- * @param $nids
- *   (optional) An array of node IDs (nid). Defaults to FALSE so empty feeds can
- *   be generated with passing an empty array, if no items are to be added
- *   to the feed.
- * @param $channel
- *   (optional) An associative array containing 'title', 'link', 'description',
- *   and other keys, to be parsed by format_rss_channel() and
- *   format_xml_elements(). A list of channel elements can be found at the
- *   @link http://cyber.law.harvard.edu/rss/rss.html RSS 2.0 Specification. @endlink
- *   The link should be an absolute URL.
- *
- * @see node_menu()
- */
-function node_feed($nids = FALSE, $channel = array()) {
```

Comment:
Hm, not sure about removing this. Definitely an API change deleting a function like this. Perhaps just mark it `@deprecated since 1.5.0`